packages/grafana-schema/src/common/data_frame.cue

```diff
@@ -0,0 +1,216 @@
+package common
```

Comment:
@sdboyer generating types for this file fails with error `failed to generate common schema TS: could not generate field: %!w(*errors.posError=&{{0xc001125180 19732} [] {bottom, unsatisfiable []}})`, started happening after I've added `@cuetsy(kind="interface")` to the types. Any idea what could it be?

Reply:
Found the issue, looks like having a separate type with a single value doesn't work, e.g. `#DecimalCount?: int64 @cuetsy(kind="type")`.